micrometer-core/src/main/java/io/micrometer/core/instrument/binder/logging/LogbackMetrics.java

```diff
@@ -130,13 +130,18 @@ public void close() {
 
     @Override
     public FilterReply decide(Marker marker, Logger logger, Level level, String format, Object[] params, Throwable t) {
+        // If format is null we can short circuit
```

Comment:
Can you add to the comment _why_ the format is null? Based on your PR description it sounds like format is null when doing an `isDebugEnabled` check. That would be useful info here in the code

Reply:
Thanks for feedback; updated with more elaborate comment!